build_tools/cmake/iree_cc_test.cmake

```diff
@@ -107,6 +107,56 @@ function(iree_cc_test)
   set_property(TARGET ${_NAME} PROPERTY DIRECT_DEPS ${_RULE_DEPS})
 
   string(REPLACE "::" "/" _PACKAGE_PATH ${_PACKAGE_NS})
+
+  # Case for cross-compiling towards Android.
+  if(ANDROID)
```

Comment:
Pretty long function now, could refactor this block into a helper to save on some indentation. WDYT?

```
if(ANDROID)
  helper_function(args...)
  return()
endif()
```

Reply:
+1. I might get to this later.